packages/server/scripts/generate-secrets.mjs

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+import crypto from "node:crypto";
+import bcrypt from "bcryptjs";
+import { intro, password, note, outro, isCancel, cancel } from "@clack/prompts";
+
+function generateJWTSecret() {
+  return crypto.randomBytes(64).toString('hex');
+}
+
+async function generatePasswordHash(password) {
+  return await bcrypt.hash(password, 12);
+}
+
+async function main() {
+  intro('🔐 Counterscale Secret Generator');
+  
+  const jwtSecret = generateJWTSecret();
+  
+  const userPassword = await password({
+    message: 'Enter password to hash:',
+    validate: (value) => {
+      if (!value.trim()) {
+        return 'Password cannot be empty';
+      }
+      if (value.trim().length < 12) {
+        return 'Password must be 12 characters or more';
+      }
+    }
+  });
+  
+  if (isCancel(userPassword)) {
+    cancel('Operation cancelled');
+    process.exit(0);
+  }
+  
+  try {
+    const passwordHash = await generatePasswordHash(userPassword);
+    
+    const output = [
+      'Copy these values to your .dev.vars file:',
```

Comment:
Having a utility script for local development – that makes sense to me.

But people just deploying Counterscale should be able to do this all in the `cli` (don't need a separate script). We prompt them for their password, then store the value as a production CF secret using wrangler commands (see the cli code for examples). In that version we don't even bother saving locally; we just push everything to prod.

I'm okay if we do that as a follow up PR.